dashboard/src/components/ui/v2/Text/Text.tsx

```diff
@@ -1,61 +1,62 @@
-import { styled } from '@mui/material';
+import { styled } from '@mui/material'
 import type {
   TypographyProps as MaterialTypographyProps,
-  TypographyTypeMap,
-} from '@mui/material/Typography';
+  TypographyTypeMap
+} from '@mui/material/Typography'
 import MaterialTypography, {
   getTypographyUtilityClass,
-  typographyClasses as materialTypographyClasses,
-} from '@mui/material/Typography';
-import clsx from 'clsx';
+  typographyClasses as materialTypographyClasses
+} from '@mui/material/Typography'
+import clsx from 'clsx'
 
 export type TextProps<
   D extends React.ElementType = TypographyTypeMap['defaultComponent'],
-  P = {},
+  P = {}
 > = MaterialTypographyProps<D, P> & {
   /**
    * The color of the text.
    *
    * @default 'primary'
    */
-  color?: 'primary' | 'secondary' | 'disabled' | 'error';
+  color?: 'primary' | 'secondary' | 'disabled' | 'error' | 'warning'
   /**
    * The component used for the root node.
    */
-  component?: D;
-};
+  component?: D
+}
 
 const textClasses = {
   ...materialTypographyClasses,
   colorPrimary: getTypographyUtilityClass('colorPrimary'),
   colorSecondary: getTypographyUtilityClass('colorSecondary'),
   colorDisabled: getTypographyUtilityClass('colorDisabled'),
   colorError: getTypographyUtilityClass('colorError'),
-};
+  colorWarning: getTypographyUtilityClass('colorWarning')
+}
 
 const StyledTypography = styled(MaterialTypography)<TextProps>(({ theme }) => ({
   color: theme.palette.text.primary,
   [`&.${textClasses.subtitle1}`]: {
-    color: theme.palette.text.secondary,
+    color: theme.palette.text.secondary
   },
   [`&.${textClasses.subtitle2}`]: {
-    color: theme.palette.text.secondary,
+    color: theme.palette.text.secondary
   },
   [`&.${textClasses.colorSecondary}`]: {
-    color: theme.palette.text.secondary,
+    color: theme.palette.text.secondary
   },
   [`&.${textClasses.colorDisabled}`]: {
-    color: theme.palette.text.disabled,
+    color: theme.palette.text.disabled
   },
   [`&.${textClasses.colorError}`]: {
-    color: theme.palette.error.main,
+    color: theme.palette.error.main
   },
-}));
+  [`&.${textClasses.colorWarning}`]: {
+    color: theme.palette.warning.main
```

Comment:
```suggestion
    color: theme.palette.warning.dark,
```

I would go with the darker color to make the text easier to read.